parmed/openmm/parameters.py

```diff
@@ -312,8 +310,48 @@ def _write_omm_cmaps(self, dest):
             if (a1, a2, a3, a4, a5) in used_torsions: continue
             used_torsions.add((a1, a2, a3, a4, a5))
             used_torsions.add((a5, a4, a3, a2, a1))
-            dest.write('   <Torsion map="%d" class1="%s" class=2"%s" '
-                       'class3="%s" class4="%s" class5="%s"/>\n' %
+            dest.write('   <Torsion map="%d" type1="%s" type=2"%s" '
+                       'type3="%s" type4="%s" type5="%s"/>\n' %
                        (maps[id(cmap)], a1, a2, a3, a4, a5)
             )
         dest.write(' </CmapTorsionForce>\n')
+
+    def _write_omm_nonbonded(self, dest):
+        if self.combining_rule == 'geometric':
+            if len(self.nbfix_types) > 0:
+                raise NotImplementedError("Nonbonded forces with geometric combining rules have not been implemented yet.")
+            else:
+                raise NotImplementedError("Nonbonded forces with geometric combining rules have not been implemented yet.")
+
+        if len(self.nbfix_types) > 0:
+            raise NotImplementedError("Nonbonded forces with geometric combining rules have not been implemented yet.")
```

Comment:
Error message is misleading here.  The incompatibility is NBFIX, not geometric combining